modules/swagger-codegen/src/main/resources/typescript-fetch/configuration.mustache

```diff
@@ -8,5 +11,6 @@ export class Configuration {
     };
     username: string;
     password: string;
-    accessToken: string;
+    accessToken: string | ((name: string) => string);
+    basePath: string;
 }
```

Comment:
Yes, this all looks cool. The point of my comment is that there's missing newline https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.